netci.groovy

```diff
@@ -10,7 +10,7 @@ def project = GithubProject
 def branch = GithubBranchName
 def isPR = true
 
-def osList = ['Windows_NT', 'Windows_NT_FullFramework', 'Ubuntu14.04']
+def osList = ['Windows_NT', 'Windows_NT_FullFramework', 'Ubuntu14.04', 'Ubuntu16.04', 'OSX']
```

Comment:
See https://github.com/dotnet/corefx/pull/16562/files for how corefx explicitly includes 10.12